scripts/docker-build-and-push.sh

```diff
@@ -19,16 +20,18 @@ Options:
   -h, --help
         Display this help and exit.
 
-  -d, --dockerfile
+  -d <PATH>, --dockerfile <PATH>
         Build and push only this Dockerfile. Otherwise all found will be used.
 
-  -r, --registry
-        List of one or more Docker registries to push to, without the trailing slash.
-        Remember to change the corresponding value in conf/resinhup.conf as well.
-        Default: "$_registry_default".
+  -p <PATHS>, --paths <PATHS>
+        List of one or more Docker registry paths to push to, quoted, separated
+	with spaces. The paths must end with the image name.
```

Comment:
A missing space here?
What do you mean by "The paths must end with the image name"?